website/docs/r/spot_fleet_request.html.markdown

```diff
@@ -78,6 +78,43 @@ resource "aws_spot_fleet_request" "foo" {
 }
 ```
 
+```hcl
+resource "aws_spot_fleet_request" "cheap_compute" {
+  iam_fleet_role                      = "arn:aws:iam::12345678:role/spot-fleet"
+  target_capacity                     = 3
+  valid_until                         = "2019-11-04T20:44:20Z"
+  allocation_strategy                 = lowestPrice
+  fleet_type                          = request
+  wait_for_fulfillment                = true
+  terminate_instances_with_expiration = true
+  
+  dynamic "launch_specification" {
+    for_each = [for s in var.subnets : {
+      subnet_id = s[1]
+    }]
+    content {
+      ami                         = "ami-1234"
+      instance_type               = "m4.4xlarge"
+      subnet_id                   = launch_specification.value.subnet_id
+      vpc_security_group_ids      = "sg-123456"
+
+      root_block_device {
+        volume_size           = "8"
+        volume_type           = "gp2"
+        delete_on_termination = "true"
+      }
+
+      tags = {
+        Name        = "Spot Node"
+        tag_builder = "builder"
+      }
+    }
+  }
+}
+```
+
+~> In this example, we use a [`dynamic` block](/docs/configuration/expressions.html#dynamic-blocks) to define zero or more `launch_specification` blocks, producing one for each element in the list of subnet ids.
```

Comment:
Hey @FernandoMiguel, I am just passing by this one, but If you want to make a blue block, use this:
```suggestion
-> In this example, we use a [`dynamic` block](/docs/configuration/expressions.html#dynamic-blocks) to define zero or more `launch_specification` blocks, producing one for each element in the list of subnet ids.
```